### PR TITLE
Rocket Launcher Damage Increase

### PR DESCRIPTION
**Pull request by harry3124**
Damage has been severely low on the previous value.
Splash damage/radius has been decreased to compensate.

### CODE REVIEW

File: code/modules/projectiles/guns/launcher/rocket.dm [hunk 1]
```diff
@@ -38,8 +38,4 @@
 		rockets -= I
 		return new I.projectile_type(src)
 	return null
-
-/obj/item/weapon/gun/launcher/rocket/handle_post_fire(mob/user, atom/target)
```
Comment (Rykka-Stormheart) [inline code/modules/projectiles/guns/launcher/rocket.dm:42]: This should be left in (not the admin log/message, just the proc) - launchers may need to do a callback post-fire, but if you're set on removing it, remove the ..() below.

**Review by Rykka-Stormheart — CHANGES_REQUESTED** [review]
The proc should be left in, admin log + messages can safely be removed, but if you're removing the entire proc, remove the ..() too

**Comment by Cyrelius** [comment]
The armour penetration might be a little much, but this looks very good overall.  My worry is one-shotting people, it's not fun when you get oneshot-killed.

**Comment by Razgriz1032** [comment]
To do that much armor pen and have a 4 tile explosion radius would be too much.

I'd honestly make multiple rocket types, HE with no armor pen and the 4 tile radius and lower damage, and HEAT with large pen and small radius and high damage.

**Comment by Rykka-Stormheart** [comment]
> To do that much armor pen and have a 4 tile explosion radius would be too much.
> 
> I'd honestly make multiple rocket types, HE with no armor pen and the 4 tile radius and lower damage, and HEAT with large pen and small radius and high damage.

This would be awesome, choose your ammo based on situation, rather than one-size-beats-all.

**Comment by Razgriz1032** [comment]
Ahh, missing comments. See this for comments guidelines. 
https://github.com/CHOMPStation2/CHOMPStation2/blob/3fe84f61e8fff3cc962c803207b2f0acf80d3576/.github/CONTRIBUTING.md

==============
When changing any code in any stock Polaris .DM file, you must mark your changes:
For single-line changes: //CHOMPEdit - "Explanation" (Edit can also be Add for new lines or Removal if you are commenting the line out)
For multi-line additions: //CHOMPEdit - "Explanation" and then at the bottom of your changes, //CHOMPEdit End
For multi-line removals: Use a block comment (/* xxx */) to comment out the existing code block (do not modify whitespace more than necessary) and at the start, it should contain /* CHOMP Removal - "Reason"
==============

**Pull request closed without merging**